test/integration_tests/long/test_model_forecasting.py

```diff
@@ -103,8 +110,6 @@ def test_forecast(self):
             TYPE Forecasting
             HORIZON 12
             PREDICT 'y'
-            LIBRARY 'neuralforecast'
-            AUTO 'false'
```

Comment:
For my education, is neuralforcast not working? There is no GPU in the circleci, that's where I am getting confused.

Reply:
I see, was not aware of that. Neuralforecast works fine mostly, except if a corresponding function is created in the same python file where other things are done as well, like creating table etc. This is a GPU-specific error, which I am able to locally reproduce. In the CPU however, Neuralforecast will take a lot of time to train.